examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,222 @@
+"""
+Functional connectivity kinds for group analysis of connectomes
+===============================================================
+
+This example compares different kinds of functional connectivity between
+regions of interest : correlation, partial correlation, as well as a kind
+called **tangent**. The resulting connectivity coefficients are used to
+discriminate ADHD patients from healthy controls and the **tangent kind**
+**outperforms** the standard connectivity kinds.
+"""
+# A useful matrix plotting function
+import numpy as np
+import matplotlib.pylab as plt
+
+
+def plot_matrices(matrices, matrix_kind):
+    n_matrices = len(matrices)
+    plt.figure(figsize=(n_matrices * 4, 4))
+    for n_subject, matrix in enumerate(matrices):
+        plt.subplot(1, n_matrices, n_subject + 1)
+        matrix = matrix.copy()  # avoid side effects
+        # Set diagonal to zero, for better visualization
+        np.fill_diagonal(matrix, 0)
+        vmax = np.max(np.abs(matrix))
+        plt.imshow(matrix, vmin=-vmax, vmax=vmax, cmap='RdBu_r',
+                   interpolation='nearest')
+        plt.title('{0}, subject {1}'.format(matrix_kind, n_subject))
+
+
+###############################################################################
+# Load ADHD dataset and MSDL atlas
+# --------------------------------
+
+###############################################################################
+# We study only 20 subjects from the ADHD dataset, to save computation time.
+from nilearn import datasets
+
+adhd_data = datasets.fetch_adhd(n_subjects=20)
+
+###############################################################################
+# We use probabilistic regions of interest (ROIs) from the MSDL atlas.
+msdl_data = datasets.fetch_atlas_msdl()
+msdl_coords = msdl_data.region_coords
+n_regions = len(msdl_coords)
+print('MSDL has {0} ROIs, part of the following networks:\n{1}.'.format(
+    n_regions, msdl_data.networks))
+
+###############################################################################
+# Region signals extraction
+# -------------------------
+
+###############################################################################
+# To extract regions time series, we instantiate a
+# :class:`nilearn.input_data.NiftiMapsMasker` object and pass the atlas the
+# file name to it, as well as filtering band-width and detrending option.
+from nilearn import input_data
+
+masker = input_data.NiftiMapsMasker(
+    msdl_data.maps, resampling_target="maps", t_r=2.5, detrend=True,
```

Comment:
Ok, I think we should also fix the doc
https://github.com/nilearn/nilearn/blob/master/nilearn/input_data/nifti_maps_masker.py#L87